examples/gallery/10_wheel_ground_contact_patch.py

```diff
@@ -0,0 +1,207 @@
+# Copyright (C) 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""
+.. _ref_contact_patch:
+
+========================================================================
+Create a contact patch for wrapping between a wheel and ground interface
+========================================================================
+
+**Summary**: This example demonstrates how to create a contact patch for use with wrapping
+to avoid meshing into a narrow contact region between two objects.
+
+Objective
+~~~~~~~~~
+This example uses a contact patch for wrapping to avoid the interface of a wheel with the ground
+to improve mesh quality when growing prism layers in the region of the contacting faces.
+
+.. image:: ../../../images/contact_patch.png
+   :align: center
+   :width: 600
+
+Image showing the following:
+* Top left: wheel/ground interface.
+* Top right: addition of contact patch.
+* Lower left: grouping tolerance at 4 with multiple contact patches.
+* Lower right: grouping tolerance at 20 with merged single contact patch.
+
+Procedure
+~~~~~~~~~
+* Launch an Ansys Prime Server instance and instantiate the meshing utilities
+  from the ``lucid`` class.
+* Import the wheel ground geometry.
+* Convert the topo parts to mesh parts so that the contact patch can be created.
+* Create a contact patch between the wheel and the ground.
+* Extract the fluid region using wrapping.
+* Volume mesh with polyhedral and prism cells.
+* Write a CAS file for use in the Fluent solver.
+* Exit the PyPrimeMesh session.

```

Comment:
```suggestion
#.  Launch an Ansys Prime Server instance and instantiate the meshing utilities
   from the ``lucid`` class.
#. Import the wheel ground geometry.
#.  Convert the topo parts to mesh parts so that the contact patch can be created.
#. Create a contact patch between the wheel and the ground.
#. Extract the fluid region using wrapping.
#. Volume mesh with polyhedral and prism cells.
#. Write a CAS file for use in the Fluent solver.
#. Exit the PyPrimeMesh session.

```
This should be a numbered list as provides sequential steps.